doc/src/manual/constructors.md

```diff
@@ -330,21 +330,22 @@ This automatic provision of constructors is equivalent to the following explicit
 julia> struct Point{T<:Real}
            x::T
            y::T
-           Point{T}(x,y) where T<:Real = new(x,y)
+           Point{T}(x,y) where {T<:Real} = new{T}(x,y)
```

Comment:
This should be `new` instead of `new{T}`, because that is allowed, and we can automatically figure out the type to construct in this case.